libs/community/langchain_community/tools/riza/command.py

```diff
@@ -0,0 +1,91 @@
+"""
+Tool implementations for the Riza (https://riza.io) code interpreter API.
+
+Documentation: https://docs.riza.io
+API keys:      https://dashboard.riza.io
+"""
+
+from typing import Any, Optional, Type
+
+from langchain_core.callbacks import (
+    CallbackManagerForToolRun,
+)
+from langchain_core.pydantic_v1 import BaseModel, Field
+from langchain_core.tools import BaseTool, ToolException
+
+
+class ExecPythonInput(BaseModel):
+    code: str = Field(description="the Python code to execute")
+
+
+class ExecPython(BaseTool):
```

Comment:
Consider adding inline documentation to the tool for users -- So it surfaces on the API reference. You can include in the doc string a code example as well